docs/connecting_to_remote_db_aws_aurora.md

```diff
@@ -0,0 +1,101 @@
+# Connecting to a remote Klaxon database in an AWS Aurora cluster
+
+This documentation is primarily for first-time setup to connect a locally running app to a remote Aurora cluster database. This doc assumes several things:
+- The app is running locally in a Docker container
+- You have already created an Aurora RDS cluster where the Klaxon database will be housed. That cluster should be configured in a way that is compatible with PostgreSQL 14. For our purposes at the Post, we used `database_engine_version: 14.4` and `instance_size: db.t4g.medium`. [Here is a guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html#Concepts.DBInstanceClass.Support) to help you select a compatible instance size. We deploy our AWS resources in infrastructure-specific repos, however resources [can also be created in the AWS console](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html).
```

Comment:
i'm wondering if we should add a note along the lines of `WaPo users: This cluster should already be set up and viewable at <link to AWS console>`

Reply:
I'm really hesitating to add anything WaPo-specific to the documentation to make it really easy to get PRs approved back to Klaxon. I was talking to Paige about this earlier and basically think we should maintain documentation that doesn't serve all Klaxon users in an internal directory (counter to our new norm around documentation)

Reply:
I say that knowing that I referenced "the Post" in the md, but my thought was that our suggestion could help other users

Reply:
cool that makes sense!